_pages/index.html

```diff
@@ -0,0 +1,114 @@
+  <body>
+    {% include header.html %}
+
+    <main>
+      <section class="card-body course-heading">
+        <h1 class="heading">
+          <img class="logo" src="images/favicons/uiuc.png"/>CS 241: An Introduction to System Programming in C
+          <p>
+            Spring 2019
+          </p>
+        </h1>
+      </section>
+      <section class="card-body course-description">
+        <h2>
+          Course Description
+        </h2>
+        This course is designed to challenge you as a programmer and new
+        computer scientist at the University of Illinois. Rather than the
+        sand-boxed, contained, and simple problems of your previous courses that
+        used significant scaffolding and pre-built libraries, you will be
+        interacting with a much more complex environment: the entire system and
+        even computing networks. Further, you will need to fully understand how
+        memory is allocated, used, and re-used within a process. You will also
+        need to know how input and output can be optionally buffered between
+        processes and files. In short, it is time to remove the training wheels
+        of and instead fling open the doors, welcoming you to the big, wide
+        world of computing. Oh, and did we mention the challenge of concurrency
+        and solving asynchronous problems, so that your program can take
+        advantage of the multi-core CPU inside each machine?
+
+      </section>
+      </section>
+        <section class="card-body course-information">
+            <section class="flexbox">
+                <section class="flexbox-item">
+                    <h2>
+                        Time and Location
+                    </h2>
+                    <p>
+                        <strong>Instructor: </strong>Lawrence Angrave (angrave)
+                    </p>
+                    <p>
+                        <strong>Lecture: </strong>MWF 8am-8.50am
+                    </p>
+                    <p>
+                    </p>
+                    <p>
+                        <strong>Location: </strong>ECEB 1002
+                    </p>
+                </section>
+                <section class="flexbox-item">
+                    <h2>
+                        Peer Tutoring
+                    </h2>
+                    <p>
+                    <p>
+                        You can find a full list of times and locations on the <a href="/peer_tutoring.html">calendar</a>.
+                    </p>
+                    <p>
+                        <a
+                        href="https://piazza.com/illinois/spring2019/cs241"
+                        class="btn btn-primary"
+                        role="button"
+                        >Piazza</a>
+                        </p>
+                        <p>
+                            Please read our <a href="http://localhost:4000/resources.html#piazza-search-first-dont-post">Piazza policy, before posting.</a>
```

Comment:
localhost?

Reply:
My bad, this should be `/resources#piazza-search-first-dont-post`.